modifiers/heading_id.js

```diff
@@ -110,6 +111,20 @@ module.exports = (md, options) => {
             token.attrs.push(idAttr);
         }
 
+        // Generate hash link if option is set
+        if (optsObj.hashLink && level <= 3) {
+            // Grab the constructor from current token
+            const Token = token.constructor;
+            // Generate tokens for hash link
+            const linkOpen = new Token('link_open', 'a', 1);
+            linkOpen.attrs = [ [ 'class', 'hash-anchor' ], [ 'href', `#${idAttr[1]}` ], [ 'aria-hidden', true ] ];
```

Comment:
Could we add a `hashLinkClass` setting that defaults to `hash-anchor`, so a user can change the class name if they wanted to? (this will also need to be a SCSS var -- a few plugins do this, like the table wrapper)